web/components/welcome/welcome.css

```diff
@@ -1,32 +1,11 @@
-:root {
-  --color-cod-gray: #1a1a1a;
-  --color-gray: #808080;
-  --color-dove-gray: #707070;
-  --color-dodger-blue: #13b3ff;
-  --color-cod-gray: #1a1a1a;
-  --color-shamrock: #2dd070;
-  --color-torch-red: #ff004f;
-}
-
-/* Reset */
-
-html,
-body {
-  margin: 0;
-  padding: 0;
-  height: 100%;
-  /* box-sizing: border-box; */
-  font-family: 'Roboto', sans-serif;
-  font-size: 1rem;
-}
-
-h1,
+@import '../config/down-arrow.css';
```

Comment:
Nice, as long as you know the difference between the @import from CSS and the @import from Sass, that's cool